EU4toV2/Source/V2World/V2World.cpp

```diff
@@ -1874,3 +2042,309 @@ std::string V2::World::clipCountryFileName(const std::string& incoming) const
 	else
 		return incoming.substr(0, 76) + ".txt";
 }
+
+
+void V2::World::copyFolder(const std::string& folder, const std::string& src, const std::string& dest) const
+{
+	if (commonItems::DoesFolderExist(src + "/" + folder))
+	{
+		fs::copy(src + "/" + folder, dest + "/" + folder, fs::copy_options::recursive);
+	}
+}
+
+void V2::World::mergeFolder(const std::string& folder, const std::string& src, const std::string& dest) const
```

Comment:
I think copyfolder already does this. it uses 
		fs::copy(fs::u8path(sourceFolder), fs::u8path(destFolder), fs::copy_options::recursive);
on the folder itself. Might test if it works for mergers.